Build/Projects/MGCB.definition

```diff
@@ -10,6 +10,8 @@
     <Reference Include="System.Xml" />
     <Reference Include="MonoGame.Framework" />
     <Reference Include="MonoGame.Framework.Content.Pipeline" />
+
+    <Reference Include="MGCB.References" />
```

Comment:
You definitely don't want to remove Linux as a platform in Protobuild; this will break anyone depending on MonoGame.Framework through Protobuild (including me).

Instead you want to leave the standard platforms (Linux, Windows, etc.) and tell them to enable the relevant services instead.

Basically, you want to have a standard set of platforms (which represent the active different environments, even if the code itself does not have to be different), and use services specific to your project (MonoGame) to select common or different sections of code based on what is needed.